README.md

```diff
@@ -140,6 +140,20 @@ linear_function:
     intercept: 1.0"
 ```
 
+#### Supported Name Handlers
```

Comment:
Thanks -- a welcome addition. If you felt motivated adding it to the docs/ would be helpful too (not necessary for merge)

Reply:
Definitely will do.